### PR TITLE
test: e2e test for registering an organisation and seeding user per test run

### PR DESCRIPTION
**Pull request by phlptt**

### CODE REVIEW

File: ui_automation_tests/conftest.py [hunk 1]
```diff
@@ -68,6 +68,13 @@
 
 
 def pytest_addoption(parser):
+    settings.configure()
```
Comment (kowalcj0) [inline ui_automation_tests/conftest.py]: @phlptt this might be causing that `DJANGO_SETTINGS_MODULE` error

File: Pipfile [hunk 2]
```diff
@@ -16,6 +16,8 @@ pytest-html = "~=2.0"
 pytest-rerunfailures = "~=8.0"
 pytest-xdist = "~=1.31"
 selenium = "~=3.141"
+directory-sso-api-client = "*"
```
Comment (Mark-Spiers-kainos) [inline Pipfile]: I thought we were setting the pipfile to exact versions

Comment (BenAndrew-Kainos) [reply Pipfile:19]: Yeah please do that

File: pipeline/Jenkinsfile [hunk 3]
```diff
@@ -15,7 +15,7 @@ spec:
     role: worker
   containers:
   - name: lite-selenium-test
-    image: quay.io/uktrade/lite-selenium-base
+    image: quay.io/uktrade/lite-selenium-base:morelibs
```
Comment (BenAndrew-Kainos) [inline pipeline/Jenkinsfile:18]: Point this back to master when you push

File: Pipfile (hunk 2 above)
Comment (phlptt) [reply Pipfile:19]: done thanks

File: ui_automation_tests/conftest.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import datetime
 import os
-
+from django.conf import settings
```
Comment (jfaracik) [inline ui_automation_tests/conftest.py:3]: Does this still refer to our frontends settings file?

**Pull request merged**